lib/src/ast/event.dart

```diff
@@ -159,7 +158,7 @@ class ParsedEventAst extends TemplateAst
   @override
   String get postfix {
     final split = _nameWithoutParentheses.split('.');
-    return split.length > 1 ? split.last : null;
+    return split.length > 1 ? split[1] : null;
```

Comment:
Sounds like a good plan. I'll likely throw the error at the scanner level since the event module has no knowledge of the exceptionHandler.